pkg/prm/docker.go

```diff
@@ -339,7 +340,7 @@ func (d *Docker) initClient() (err error) {
 			return err
 		}
 
-		ctx, cancel := context.WithTimeout(context.Background(), 30*time.Minute)
+		ctx, cancel := context.WithTimeout(context.Background(), (d.ContextTimeout))
```

Comment:
This is the file and code path I think we want to add a unit test for, to see what happens if we pass an arbitrarily short timeout (ie it should end with the timeout killing the process).

Reply:
Unable to implement unit test's as the relevant function is set to private, as such it will be left for acceptance test's which are not et implemented.